airbyte-integrations/connectors/tasks.py

```diff
@@ -94,13 +96,26 @@ def _run_single_connector_task(args: Iterable) -> int:
     return _run_task(*args)
 
 
-def _run_task(ctx: Context, connector_string: str, task_name: str, multi_envs: bool = True, **kwargs: Any) -> int:
+def _run_task(
+    ctx: Context,
+    connector_string: str,
+    task_name: str,
+    multi_envs: bool = True,
+    module_path: Optional[str] = None,
+    task_commands: Dict = TASK_COMMANDS,
+    **kwargs: Any,
+) -> int:
     """
     Run task in its own environment.
     """
+    cur_dir = os.getcwd()
     if multi_envs:
-        source_path = f"source_{connector_string.replace('-', '_')}"
-        os.chdir(os.path.join(CONNECTORS_DIR, f"source-{connector_string}"))
+        if module_path:
+            os.chdir(module_path)
+            source_path = connector_string
+        else:
+            os.chdir(os.path.join(CONNECTORS_DIR, f"source-{connector_string}"))
+            source_path = f"source_{connector_string.replace('-', '_')}"
```

Comment:
will it work with destinations' connectors?

Reply:
`tasks.py` is used to check connectors with pre-commit hook.
I've made required fixes, so we could use it for any module. 
So these 2 lines are not executed when called from `ci_static_check_reports `